Implement copy button

aushaag opened the pull request:
Create new copy button in the symbol output (disabled by default). When users click the copy button, copy the labels of the selected symbols to the clipboard:

![image](https://user-images.githubusercontent.com/23203007/114239983-ce289200-9954-11eb-8663-0642ae231dd7.png)

Disable CopyButton by default, and give users the option to enable it in Settings/Navigation/ShowCopyButton:

![image](https://user-images.githubusercontent.com/23203007/114239885-a76a5b80-9954-11eb-8c27-a74ddbfb35d2.png)

aushaag committed:
Changing "Navigation" to "Navigation and Buttons" "Buttons" for some of the translations

Since there are button settings in the "Navigation" section of buttons, it seems clearer to rename "Navigation" in the settings list to "Navigation and Buttons". There are many languages I have not added this to yet, but I can complete the translations soon.

aushaag committed:
Add copy button to settings

Add new setting for showing/hiding the copy button in the output. Also add dividers between options to match other settings.

aushaag committed:
Implement Copy Button

Implement copy button functionality. When copy button is clicked, output labels of currently selected symbols to clipboard

aushaag committed:
Create new CopyButton Component

Create CopyButton component that follows BackspaceButton as template.

aushaag committed:
Add CopyButton to SymbolOutput

Add CopyButton to SymbolOutput to the left of backspace and clear. CopyButton is only displayed when enabled in navigationSettings (copyShowActive). CopyButton is also only displayed when at least one symbol has been selected (symbols.length > 0).

aushaag committed:
Add CopyShow setting message

Add text to display for the setting that enables/disables showing the CopyButton. Note: these messages need translations!

martinbedouret reviewed (changes requested):
@aushaag Terrific job!! Congrats! 
My only comment is that you don't need to worry about the translation files. You only need to add any new string to be translated into the `cboard\src\translations\src\cboard.json` file.  before the production release, an automation script will take care of the translations.

aushaag committed:
Remove brackets for one-line map

Curly brackets are unnecessary

aushaag committed:
Update cboard.json to include copy button text for settings

These are the descriptions next to the toggle to enable/disable the copy button in the output panel

aushaag committed:
Unnecessary: fix my typo in Navigation.messages.js

I believe this will be overridden by the translation script anyway, but I realized there was a typo and "Display" should be "Navigation".

sylvansson reviewed (commented):
Thanks for the great first contribution @aushaag :-)
